styles.css

```diff
@@ -65,3 +70,49 @@
   padding-top: 5px;
   padding-bottom: 10px;
  }
+
+.shadow-add {
+  position: relative;
+  box-shadow: 10px 5px 10px 10px rgba(0,0,0,0.2);
+ }
+
+.form-container {
+  background-color: #f5c85f;
+  display: flex;
+  padding-top: 15px;
+  padding-bottom: 20px;
+  align-items: center;
+  flex-direction: column;
+ }
+ 
+.footer-title {
+  font-size: 16px;
+  color: #EEE;
+ }
+ 
+.footer-box[type=text] {
+  padding:5px 25px;
+  color: #000;
+  width: 250px;
+  height: 30px;
+  margin-top: 10px;  
+  margin-bottom: 20px;  
+  border-radius: 10px;
+  border: none;
+ }
+
+input#footer-botton { 
```

Comment:
don't use ids to target elements in css, also don't use HTML elements for that too,  just add a class